apps/crn-server/src/app.ts

```diff
@@ -177,8 +194,14 @@ export const appFactory = (libs: Libs = {}): Express => {
   const groupDataProvider =
     libs.groupDataProvider ||
     new GroupSquidexDataProvider(squidexGraphqlClient);
-  const newsDataProvider =
+  const newsSquidexDataProvider =
     libs.newsDataProvider || new NewsSquidexDataProvider(newsRestClient);
+  /* istanbul ignore next */
```

Comment:
I am ignoring this line for now, the only way I thought about testing this was with an integration test. What do you think. If we follow this path, then some more work needs to be done in CI I believe to create the env in contentful